### PR TITLE
datadog: Add missing priority

### PR DESCRIPTION
**Pull request by flysen**
##### SUMMARY
Add a missing property for priority in datadog monitor module

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
datadog_monitor
`priority=module.params['priority'],`

### CODE REVIEW

File: plugins/modules/monitoring/datadog/datadog_monitor.py [hunk 1]
```diff
@@ -153,6 +153,10 @@
         type: bool
         default: yes
         version_added: 1.3.0
+    priority:
+        description:
+          - Integer from 1 (high) to 5 (low) indicating alert severity.
+        type: str
```
Comment (felixfontein) [inline plugins/modules/monitoring/datadog/datadog_monitor.py:159]:
```suggestion
        type: int
```
If it is an integer, this should have `type=int`. Also if there are just five possibilities, it's better to add `choices`:
```suggestion
        type: str
        choices: [1, 2, 3, 4, 5]
```

File: plugins/modules/monitoring/datadog/datadog_monitor.py [hunk 2]
```diff
@@ -239,6 +243,7 @@ def main():
             evaluation_delay=dict(),
             id=dict(),
             include_tags=dict(required=False, default=True, type='bool'),
+            priority=dict(),
```
Comment (felixfontein) [inline plugins/modules/monitoring/datadog/datadog_monitor.py:246]:
```suggestion
            priority=dict(type='int'),
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

File: plugins/modules/monitoring/datadog/datadog_monitor.py (hunk 1 above)
Comment (flysen) [reply plugins/modules/monitoring/datadog/datadog_monitor.py]: Let's not declare choices. Better to follow your suggestion to keep as int and then the DD API or the DD documentation take care of choices.

File: plugins/modules/monitoring/datadog/datadog_monitor.py [hunk 3]
```diff
@@ -153,6 +153,10 @@
         type: bool
         default: yes
         version_added: 1.3.0
+    priority:
+        description:
+          - Integer from 1 (high) to 5 (low) indicating alert severity.
+        type: int
```
Comment (felixfontein) [inline plugins/modules/monitoring/datadog/datadog_monitor.py:159]:
```suggestion
        type: int
        version_added: 4.6.0
```

**Review by felixfontein — COMMENTED** [review]
You still need to add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment). Thanks.

**Comment by felixfontein** [comment]
@Tyler-2 @shermdog @albertvaka @twmartin you contributed to this module earlier. Maybe you can take a look at this change and review it? Would be appreciated :)

**Review by albertvaka — APPROVED** [review]
LGTM ✅ I can't merge though.

**Comment by felixfontein** [comment]
@albertvaka thanks! no problem, I'll press the big green button on Monday if nobody complains til then :)

**Comment by felixfontein** [comment]
@flysen thanks for your contribution!
@albertvaka thanks for reviewing!

**Pull request merged**